src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+
+import classNames from 'classnames';
+import { getSearchWith } from '../../utils/searchHelper';
+import { Gender } from '../../types/Gender';
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const CENTURIES_LINKS = ['16', '17', '18', '19', '20'];
+  const SEX_FILTER_DATA = {
+    All: null,
+    Male: 'm',
+    Female: 'f',
+  };
+
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') as Gender || null;
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const setSearchWith = (key: string, update: string | null) => {
+    setSearchParams(getSearchWith(searchParams, ({ [key]: update })));
+  };
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setSearchWith('query', event.target.value);
+  };
+
+  const handleCenturyChange = (century: string) => {
+    const filteredCenturies = centuries.includes(century)
+      ? centuries.filter(item => item !== century)
+      : [...centuries, century];
+
+    return getSearchWith(searchParams, { centuries: filteredCenturies });
+  };
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        {Object.keys(SEX_FILTER_DATA).map(key => {
```

Comment:
You can use directly `Object.values` for getting `SEX_FILTER_DATA[key]` as simple `value`

Reply:
I need key and value in this case. Although, I suppose it would be a great idea touse `Object.entries`